libshpool/src/daemon/prompt.rs

```diff
@@ -68,9 +68,9 @@ pub fn maybe_inject_prefix(
                SHPOOL__OLD_PS1="${{PS1}}"
```

Comment:
(This comment is really for line 67).

I think the array copy on line 67 is still broken, so this won't work. It should be

```
SHPOOL__OLD_PROMPT_COMMAND="${{PROMPT_COMMAND[@]}}"
```

Here is a shell session showing what I mean:

```
$ ARR1=("foo bar" "bim bap")
$ echo ${ARR1[0]}
foo bar
$ echo ${ARR1[1]}
bim bap
$ ARR2="${ARR1}" # what we do now, bad
$ echo ${ARR2[0]}
foo bar
$ echo ${ARR2[1]} # fails

$ ARR2=("${ARR1[@]}") # what we should do
$ echo ${ARR2[0]}
foo bar
$ echo ${ARR2[1]}
bim bap
```

Reply:
Ah, my bad, totally misunderstood that before. Makes sense now. Updated!